jenkins/PerfCI_Chaos/04_PerfCI_Chaos_Upgrade_OpenShift_Deployment/Jenkinsfile

```diff
@@ -322,8 +572,9 @@ END
             }
         }
         success {
-            echo 'Triggering 05-PerfCI-Chaos-Verify-Windows-VMs-Deployment pipeline'
-            build job: '05-PerfCI-Chaos-Verify-Windows-VMs-Deployment', wait: false
+            sh """ sudo podman run --rm -e KUBEADMIN_PASSWORD='${KUBEADMIN_PASSWORD}' -e ELASTICSEARCH='${ELASTICSEARCH}' -e ELASTICSEARCH_PORT='${ELASTICSEARCH_PORT}' -e CI_STATUS='pass' -e UPGRADE_MASTERS_DURATION_SECONDS='${upgradeMastersDurationInSeconds}' -e UPGRADE_WORKERS_DURATION_SECONDS='${upgradeWorkersDurationInSeconds}' -e RUN_TYPE='${RUN_TYPE}' -e log_level='INFO' -v '${KUBECONFIG_PATH}:${KUBECONFIG_PATH}' --privileged '${QUAY_BENCHMARK_RUNNER_REPOSITORY}' """
```

Comment:
Why the triple doublequote syntax?  You don't have doublequotes in the quoted string.

Reply:
Fixed